src/Zypper.cc

```diff
@@ -1918,7 +1935,8 @@ void Zypper::processCommandOptions()
     )
     .optionSectionCommandOptions()
     .option_SERVICE_PROP
-    .option( "-t, --type <TYPE>",	(str::Format(_("Type of the service (%1%).") ) % "RIS").str() )	// FIXME: leagcy, actually autodetected but check libzypp
+    .legacyOptionSection()
+    .option( "-t, --type <TYPE>",	(str::Format(_("Type of the service (%1%). This option is ignored.") ) % "RIS").str() )	// FIXME: leagcy, actually autodetected but check libzypp
```

Comment:
I wouldn't even repeat the old description, just `_("The type of service is always autodetected. This option is ignored.")

Reply:
Done